README.md

```diff
@@ -20,97 +85,114 @@ python3.10 -m venv .ovxai
+In this example, we are trying to know the reason why the model outputs a `cheetah` label for the given input image.
 
 ```python
 import cv2
 import numpy as np
 import openvino.runtime as ov
-
 import openvino_xai as xai
-from openvino_xai.explainer.explanation_parameters import ExplanationParameters
 
+# Load the model
+ov_model: ov.Model = ov.Core().read_model("mobilenet_v3.xml")
 
-def preprocess_fn(x: np.ndarray) -> np.ndarray:
-    # Implementing own pre-process function based on model's implementation
-    x = cv2.resize(src=x, dsize=(224, 224))
-    x = np.expand_dims(x, 0)
-    return x
+# Load the image to be analyzed
+image: np.ndarray = cv2.imread("tests/assets/cheetah_person.jpg")
+image = cv2.resize(image, dsize=(224, 224))
+image = np.expand_dims(image, 0)
 
-
-# Creating model
-model = ov.Core().read_model("path/to/model.xml")  # type: ov.Model
-
-# Explainer object will prepare and load the model once in the beginning
+# Create the Explainer object
 explainer = xai.Explainer(
-    model,
+    model=ov_model,
     task=xai.Task.CLASSIFICATION,
-    preprocess_fn=preprocess_fn,
 )
 
-# Generate and process saliency maps (as many as required, sequentially)
-image = cv2.imread("path/to/image.jpg")
-explanation_parameters = ExplanationParameters(
-    target_explain_labels=[11, 14],  # indices or string labels to explain
+# Generate saliency map for the label of interest
+explanation: xai.Explanation = explainer(
+    data=image,
+    targets=293,  # (cheetah), accepts label indices or actual label names
+    overlay=True,  # saliency map overlay over the input image, defaults to False
 )
-explanation = explainer(image, explanation_parameters)
-
-explanation: Explanation
-explanation.saliency_map: Dict[int: np.ndarray]  # key - class id, value - processed saliency map e.g. 354x500x3
 
-# Saving saliency maps
-explanation.save("output_path", "name")
+# Save saliency maps to output directory
+explanation.save(dir_path="./output")
 ```
 
-See more usage scenarios in [user-guide.md](docs/source/user-guide.md) and [examples](./examples).
+Original image | Explained image
+---------------|----------------
+![Oringinal images](tests/assets/cheetah_person.jpg) | ![Explained image](docs/source/_static/xai-cheetah.png)
 
-### Running example scripts
+We can see that model is focusing on the body or skin area of the animals to tell if this image contains actual cheetahs.
 
-```python
-# Retrieve OTX models by running tests
+### More advanced use-cases
+
+Users could tweak the basic use-case according to their purpose, which include but not limited to:
+
+* Select XAI mode (White-Box or Black-Box) or even specific method which are automatically decided by default
+* Provide custom model pre/post processing functions like resize and normalizations which the model expects
+* Customize output image visualization options
+* Explain multiple class targets, passing them as label indices or as actual label names
+* Explain multiple images
```

Comment:
We actually do not natively support multiple images, the user should call explainer multiple times. I would reformulate this:
```suggestion
* Call explainer multiple times to explain multiple images or to use different targets
```